### PR TITLE
b3-archive-page

### PR DESCRIPTION
**Pull request by sumitk-singh**

### CODE REVIEW

File: src/components/icon components/FilterIcon.jsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      currentPriority: "",
+      sortBy: "",
+    }));
+  };
+  return (
+    <>
+        <div id="filter-ctn">
+          <div className="flex-align-center just-con-sp-bt">
+            <h4>Filter by</h4>
+
```
Comment (zoyatanweer) [inline src/components/icon components/FilterIcon.jsx:36]: remove space

File: src/pages/Profile/Profile.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
+.profile-ctn {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+  padding: 10px;
+  width: 100%;
+  max-width: 31.25rem;
+  margin: 1rem auto;
+  background-color: #0172FF;
```
Comment (zoyatanweer) [inline src/pages/Profile/Profile.css:10]: use var for color

**Review by zoyatanweer — APPROVED** [review]
- great and clean code. 
- good to merge.

File: src/components/icon components/FilterIcon.jsx (hunk 1 above)
Comment (sumitk-singh) [reply src/components/icon components/FilterIcon.jsx:36]: ok

**Pull request merged**